src/views/RasterPlot/TimeScrollView/TimeScrollView.tsx

```diff
@@ -444,55 +448,66 @@ const TimeScrollView = <T extends {[key: string]: any}> (props: TimeScrollViewPr
         ;(divRef?.current as any)['_hasFocus'] = false
     }
 
+    const content = (
+        <div
+            style={{width: width - toolbarWidth, height, position: 'relative'}}
+            onWheel={handleWheel}
+            // onClick={handleClick}
+            onMouseDown={handleMouseDown}
+            onMouseUp={handleMouseUp}
+            onMouseMove={handleMouseMove}
+            onMouseOut={handleMouseLeave}
+        >
+            <TSVAxesLayer<T>
+                width={width - toolbarWidth}
+                height={height}
+                panels={panels}
+                panelHeight={panelHeight}
+                perPanelOffset={perPanelOffset}
+                selectedPanelKeys={selectedPanelKeys}
+                timeRange={timeRange}
+                timeTicks={timeTicks}
+                margins={definedMargins}
+                hideTimeAxis={hideTimeAxis}
+            />
+            <TSVMainLayer<T>
+                width={width - toolbarWidth}
+                height={height}
+                panels={panels}
+                panelHeight={panelHeight}
+                perPanelOffset={perPanelOffset}
+                margins={definedMargins}
+            />
+            <TSVCursorLayer
+                width={width - toolbarWidth}
+                height={height}
+                timeRange={timeRange}
+                margins={definedMargins}
+                focusTimePixels={focusTimeInPixels}
+            />
+        </div>
+    )
+
+    if (hideToolbar) {
+        <div ref={divRef}>
+            {content}
+        </div>
+    }
+
     return (
         <Splitter
             ref={divRef}
             width={width}
             height={height}
-            initialPosition={DefaultToolbarWidth}
+            initialPosition={toolbarWidth}
             adjustable={false}
         >
             <ViewToolbar
-                width={DefaultToolbarWidth}
+                width={toolbarWidth}
                 height={height}
                 customActions={timeControlActions}
             />
```

Comment:
Does this mean the toolbar `div` still gets rendered into the DOM, even if the toolbar is hidden? Is that to make sure the keyboard/mouse controls still work, or something like that?

Reply:
Seems to me kind of like we'd want to just ditch the whole `Splitter` if there's no toolbar, but I haven't really thought about it at all.

Reply:
if you look at the code above the highlighted lines, you'll see that the splitter is indeed bypassed when toolbar is hidden.

Reply:
I saw that the div ref is being used for the content, but it looked like the return still includes the splitter & toolbar divs--I wasn't quite sure how React would handle the situation where multiple divs use the same ref.

So I checked the live version and it looks like the splitter is still getting rendered with a toolbar child as well as the plot, it's just the toolbar width gets set to 0.

This doesn't seem to affect anything functionally, it just makes the page code more complicated--do we care?

Reply:
Oops you're right. I forgot to include the "return" keyword in the case of hiding the toolbar. Will fix.